README.md

```diff
@@ -1,101 +1,53 @@
-# HTML & CSS3 & JavaScript Course
+# capstone-1
 
-If you are not familiar with linters, read [root level README](../README.md).
+A website built for a local tech startup
 
-## Set-up GitHub Actions
+## Built With
 
-Please do the following **steps in this order**:
+- HTML, CSS, JS
 
-1. In the first commit of your feature branch create a `.github/workflows` folder and add a copy of [`.github/workflows/linters.yml`](.github/workflows/linters.yml) to that folder.
-   - **Remember** to use the file linked above
-   - **Remember** that `.github` folder starts with a dot.
-2. **Do not make any changes in config files - they represent style guidelines that you share with your team - which is a group of all Microverse students.**
-   - If you think that change is necessary - open a [Pull Request in this repository](../README.md#contributing) and let your code reviewer know about it.
-3. When you open your first pull request you should see the result of the GitHub actions checks:
+## Live Demo
 
-![gh actions checks](../assets/images/gh-actions-html-css-checks.png)
+You can checkout the live version [here](https://milen-ale.github.io/capstone/)
```

Comment:
- [ ] The live demo link is broken. Kindly fix this so that your project is complete.